conf/cluster_redefined.yaml.sample

```diff
@@ -0,0 +1,100 @@
+cluster:
```

Comment:
Why use cluster_defined.yaml ? Why not update cluster.yaml itself ?

Reply:
Done